samples/msal-node-samples/standalone-samples/device-code/index.js

```diff
@@ -18,6 +18,7 @@ const deviceCodeRequest = {
     deviceCodeCallback: (response) => (console.log(response.message)),
     scopes: ["user.read"],
     timeout: 20,
+    cancel: false,
```

Comment:
So how do we make cancel=true? What is the usage? When does an app know it should not poll anymore?

Reply:
Cancel is designed to be set `true` after executing this request, on this sample you can find that scenario [here](https://github.com/AzureAD/microsoft-authentication-library-for-js/blob/74f2dfce8804e25035cd3ba21d195530a3fba450/samples/msal-node-samples/standalone-samples/device-code/index.js#L30) where we set a timeout to set the `cancel` to `true` after a few seconds.

Reply:
The usage is when the request is made and we are polling, at any point after polling has begun the user of the library can program a cancel by changing the flag from `false` to `true`